provision/testdata/list_flavors.stdout.txt

```diff
@@ -19,3 +19,4 @@ INFO: kubernetes-1.22:	Kubernetes 1.22
 INFO: kubernetes-1.21:	Kubernetes 1.21
 INFO: kubernetes-1.20:	Kubernetes 1.20
 INFO: RKE-1.2.3:	Rancher Kubernetes Engine min version 1.2.3
+INFO: RKE-1.3.13:	Rancher Kubernetes Engine min version 1.3.13
```

Comment:
the latest version should be above the older versions (please see how its being done for other flavors)

Reply:
addressed

Reply:
addressed